src/internal/m365/collection/exchange/transform.go

```diff
@@ -58,6 +58,28 @@ func toMessage(orig models.Messageable) models.Messageable {
 	return CloneMessageableFields(orig, message)
 }
 
+// eventAdditionalUnneededFields contains a set of fields that don't have API
+// support right now but are ignored by or may cause errors in M365 when
+// restoring items.
+var eventAdditionalUnneededFields = []string{
+	// Will cause errors about needing to put extension data in their own requests
+	// if not removed.
+	"iCalUId_v2",
+	// Appears to be duplicate of the iCalUId.
+	"uid",
+	// Navigation links from the calendar to the event itself.
+	"calendar@odata.associationLink",
+	"calendar@odata.navigationLink",
+	// Seems like info about the request that generated the data response.
+	"@odata.context",
+	// Appears to be similar to the change tag.
+	"@odata.etag",
+	// Remove exceptions for recurring events. These will be present in objects
```

Comment:
Should we create an issue to track this future change?

Reply:
I believe recurring events code is already in main